src/Xrefcheck/Verify.hs

```diff
@@ -160,7 +159,7 @@ instance Given ColorMode => Buildable VerifyError where
       | otherwise ->
         [int||
         ⛀  Anchor '#{anchor}' is not present, did you mean:
-        #{indentF 4 $ blockListF similar}
+        #{indentF 4 $ blockListF similar}\
```

Comment:
We're also using `indentF` in line 183 to print `ExternalResourceUriConversionError` errors.

Should that also have a trailing `\` to "trim" the trailing newline?

Reply:
In fact, looking at our usage of `indentF`, it seems like we (almost?) never want it to print the trailing newline :thinking: 

How about we write a helper function with our desired behavior, e.g. `indentFWithoutNewline`, and use that instead?

Reply:
not identF but blockListF  generate newlines. In https://github.com/serokell/xrefcheck/pull/214 I already introduced new blockList function, imho it's better to come back to this when looking at #214